pcap/pcap.go

```diff
@@ -775,6 +775,10 @@ func findalladdresses(addresses *_Ctype_struct_pcap_addr) (retval []InterfaceAdd
 }
 
 func sockaddrToIP(rsa *syscall.RawSockaddr) (IP []byte, err error) {
+	if unsafe.Pointer(rsa) == nil {
```

Comment:
any reason why unsafe.Pointer is necessary here?  I'd assume that a simpler `if rsa == nil` would work just as well?

Reply:
did we get the answer for question above?

Reply:
Florian responded via email:

```
Hi!

I needed to cast rsa otherwise it paniced and crashed for me.
As I don't have access to github during office hours, I'm writting this way.

Cheers,
   Florian
```

Reply:
I a sorry why we should cast it? I did it without casting in #365  , worked for me and passed the tests.